script.js

```diff
@@ -0,0 +1,145 @@
+const projectCards = [
+  {
+    image: 'images/music.png',
+    cardTitle: 'Music Night',
+    cardDetail: ' Enjoy live show full of your favourite songs.',
+  },
+  {
+    image: 'images/dance.png',
+    cardTitle: 'Dancing',
+    cardDetail: ' Dance on music and enjoy live show full of your favourite songs.',
+  },
+  {
+    image: 'images/collab.png',
+    cardTitle: 'Collaboration',
+    cardDetail: 'You will get a chance to collaborate with singers.',
+  },
+  {
+    image: 'images/videoconference.png',
+    cardTitle: 'Meetup',
+    cardDetail: 'You will get a chance to meet with your favoratie singers.',
+  },
+];
+
+const card = document.querySelector('#allcardS');
+
+for (let i = 0; i < projectCards.length; i += 1) {
+  card.innerHTML += `<div class="Card">
+                          <img src="${projectCards[i].image}" alt="card-icon">
+                          <h5>${projectCards[i].cardTitle}</h5>
+                         <p>
+                             ${projectCards[i].cardDetail}
+                         </p>
+                        </div>`;
+}
```

Comment:
I would recommend making each `projectCard` an `article` instead of `div`. That would make more sense semantically and search engines would thank you.

Reply:
- [ Optional ] Hello @muneebulrehman As suggested b y the previous reviewer for the purpose of using the correct semantic, Kindly I recommend you to use` article` instead of `div `tag.

Reply:
I did not get it...this dynamic div is inside the article only.